spec.bs

```diff
@@ -2436,6 +2436,45 @@ in the [[#nested-traversables-intro]].
   1. Return |navigables|.
 </div>
 
+<div algorithm="inclusive-ancestor-navigables-patch">
+  Modify the [=Document/inclusive ancestor navigables=] algorithm to take a new optional
```

Comment:
We don't call this anywhere, right?

Reply:
At the moment, no. I think it's worth keeping this in place so it will always correctly invoke "ancestor navigables", and we won't need to remember to patch it later if we need to use it.

Reply:
Eh sorry, personally I don't think we should clutter the blame history with this if we're not using it. Let's introduce it when we need it, alright?

Reply:
Ok. I'll remove it.